js/ui/load_indicator.js

```diff
@@ -160,28 +160,6 @@ const LoadIndicator = Widget.inherit({
 
     _renderDimensions: function() {
         this.callBase();
-        this._updateContentSizeForAnimation();
```

Comment:
imho, we should not remove this function. It will be called after `width` runtime option change for example